app/models/concerns/wakes/metrics/facebook.rb

```diff
@@ -15,6 +15,15 @@ def enqueue_facebook_count_update
         def self.ordered_for_facebook_updates
           order("document->'facebook_count_updated_at' ASC NULLS FIRST")
         end
+
+        def update_facebook_count(count)
+          if facebook_count.to_i <= count.to_i
```

Comment:
@gaganawhad 2 things:

1. can you rename `count` to something like `new_facebook_count`? Given you compare it with the old count, having the names be very similar will communicate more clearly what's going on.

2. can you flip this around? You are testing the new count, not the old count. Think of `<=` as a method call. Seems more clear to me to call that method on the new candidate count, not the old one.

So, all together: `if new_facebook_count.to_i >= facebook_count.to_i`